### ISSUE TITLE
The MOT17 benchmark results in Figure 4 of the paper.

### ISSUE THREAD

**Issue by lzzppp** [bytedance/ColTrack#7]
Thank you for your contribution to the open source community. I encountered some problems when conducting my thesis experiments. I arranged the Coltrack experiment according to the training steps in the ReadMe. I performed pre-training based on the configuration file **"single_class.py"** in the TBD folder in the config directory. I trained for **30** epochs, and then used this weight to train the MOT task. The training configuration file used the E2E folder. **"e2e_submit_mot17.py"** configuration file. After **60** epochs of training, I used the best round results and the last round results to compare with ByteTrack. The configuration file used was **"e2e_submit_mot17.py"** in the E2E folder, but two lines were added: **just_inference_flag = True** and **num_classes = 91**. Under the frame interval of "1 2 3 6 10 15 30", the results of coltrack did not reach the results reported in the subgraph of Figure 4(a) of the paper. I followed the suggestions given by ByteTrack to modify the three parameters of the configuration file: **score_thresh**, **filter_score_thresh** and **miss_tolerance**, but it did not help.

**Comment by lzzppp** [comment]
The following is my experimental environment:
NVIDIA RTX A6000*8
Driver Version: 535.129.03
CUDA Version: 12.2

**Comment by lzzppp** [comment]
Training log
[Uploading info.txt.zip…]()

**Comment by lzzppp** [comment]
Inference settings: I saved the val data set of mot17 to a directory in demo format at different intervals. The frame id was remapped, and they are all in the format of 1 2 3 4..., so the **inference_sampler_interval** is set to 1 during inference.

**Comment by lzzppp** [comment]
Below are the experimental results of adjusting several key inference parameters.
![HOTA](https://github.com/bytedance/ColTrack/assets/67220779/90123b45-235c-449b-8d77-2e1b301daa83)
![IDF1](https://github.com/bytedance/ColTrack/assets/67220779/33a3256e-0274-447a-bb9a-80d36a501c8e)
![MOTA](https://github.com/bytedance/ColTrack/assets/67220779/3922e7dd-b5b2-4582-9f72-4b7bb2696ffe)

**Comment by lzzppp** [comment]
This is the losses:
![image](https://github.com/bytedance/ColTrack/assets/67220779/2114f988-d7f5-4b17-8e56-46d7a7d69fd1)

**Comment by lzzppp** [comment]
The HOTA metric in the training:
![image](https://github.com/bytedance/ColTrack/assets/67220779/d0351604-22fd-48c7-922d-5b0072687d2d)

**Comment by lzzppp** [comment]
I have fixed my error
